runtime/rococo/src/xcm_config.rs

```diff
@@ -77,7 +78,7 @@ type LocalOriginConverter = (
 
 parameter_types! {
 	/// The amount of weight an XCM operation takes. This is a safe overestimate.
-	pub const BaseXcmWeight: u64 = 1_000_000_000;
+	pub const BaseXcmWeight: Weight = Weight::from_ref_time(1_000_000_000);
```

Comment:
I'd like to get rid of this completely, but if it stays, it should use `from_parts` with a conservative but sensible proof-size component.